toolchain/c-toolchain.lisp

```diff
@@ -223,12 +220,21 @@
           *linkkit-start* nil
           *linkkit-end* nil)))
 
+(defun common-toolchain-parameters ()
+  (setf *cc-flags*
+        (append
+         *cc-flags*
+         ;; For MacPorts
+         #+darwin (list "-I" "/opt/local/include/")
```

Comment:
If you change this to `#+(darwin openbsd freebsd) (list "-I" "/opt/local/include/")` then you can eliminate the freebsd include above (line 214) and also then lines 35-42 in [libffi/libffi-types.lisp](https://github.com/cffi/cffi/blob/86836561ffd173cfa7a9ea2d9bd26dd6aa49b614/libffi/libffi-types.lisp#L35)

Just a thought.

Reply:
Thanks, done. (Used `/usr/local/include/` for BSD.)